prow/plugins/assign/assign.go

```diff
@@ -98,6 +106,21 @@ func parseLogins(text string) []string {
 	return parts
 }
 
+// Check if members are member of org.
+// Return a list of non-org members
+func CheckMembership(gc githubClient, log *logrus.Entry, org string, members []string) []string {
```

Comment:
```go
// checkMembership does blablabla...
func checkMembership(gc githubClient, org string, members []string) ([]string, error) {
```
Propagate up the error, don't log it out here. Function comments in go start with the name of the function.